README.md

```
@@ -1,19 +1,119 @@
+
+### Monitoring & Tools
+![Git](https://skillicons.dev/icons?i=git)
+![GitHub](https://skillicons.dev/icons?i=github)
+![Linux](https://skillicons.dev/icons?i=linux)
+![Nginx](https://skillicons.dev/icons?i=nginx)
+
+## 📊 GitHub Stats
+
+<div align="center">
+  
+  ![](https://github-readme-stats.vercel.app/api?username=zukizukizuki&show_icons=true&theme=tokyonight&count_private=true)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)